app/views/sign_up/completions/show.html.slim

```diff
@@ -1,10 +1,10 @@
--title t("titles.loa3_verified.#{session[:sp][:loa3]}", app: APP_NAME)
+-title t("titles.loa3_verified.#{session['sp][:loa3']}", app: APP_NAME)
```

Comment:
Where I could use some help is figuring out how to best convert the complex objects in IdV into simpler ones that can be serialized and deserialized with JSON.